tests/integration/Metrics/Transports/OpenTelemetryMeterProviderFactoryTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\OpenTelemetry\Tests\Integration\Metrics\Transports;
+
+use OpenTelemetry\SDK\Metrics\MeterProvider;
+use OpenTelemetry\SDK\Metrics\NoopMeterProvider;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\OpenTelemetry\Metrics\Transports\OpenTelemetryMeterProviderFactory;
+
+#[CoversClass(OpenTelemetryMeterProviderFactory::class)]
+class OpenTelemetryMeterProviderFactoryTest extends TestCase
+{
+    public function testSdkDisabledNoOpMeter(): void
+    {
+        putenv('OTEL_SDK_DISABLED=true');
```

Comment:
cleaning those up between tests can be a pain. I suggest adding: https://github.com/jakzal/phpunit-globals as a dependency

Reply:
Done